### PR TITLE
Implement batch consuming

### PR DESCRIPTION
**Pull request by Abdulsametileri**

### CODE REVIEW

**Review by mhmtszr — APPROVED** [review]
LGTM. Good feature.

File: README.md
Comment (emreodabas) [inline README.md:1]: Batch Option and Grafana & Prometheus parts could be below of the configuration part.

Comment (Abdulsametileri) [reply README.md:1]: all of them within the examples section, what is the purpose of moving it? :D

**Comment by emreodabas** [comment]
LGTM

**Pull request merged**